bot/plugins/chatrank.py

```diff
@@ -8,7 +8,9 @@
 import os
 import re
 import urllib.request
+from typing import Any
 from typing import Counter
+from typing import DefaultDict
```

Comment:
very rarely do you actually need `DefaultDict` since it is a `dict` subclass and doesn't really add too much in the way of api shape